lib/plugin.js

```diff
@@ -1,7 +1,7 @@
 import Vue from 'vue'
 import bugsnag from '@bugsnag/js'
 import bugsnagVue from '@bugsnag/plugin-vue'
-const bugsnagClient = bugsnag('<%= options %>')
+const bugsnagClient = bugsnag(<%= JSON.stringify(options) %>)
```

Comment:
options.apiKey

Reply:
But this would change the behaviour. So you cannot add more bugsnag options